### ISSUE TITLE
Execution failed for task `:flutter_glyph_kit:compileDebugKotlin`.

### ISSUE THREAD

**Issue by EverythingSuckz** [francescovallone/flutter-glyph-kit#2]
Hey, I just tried to run the example code and I got this:

```
Launching lib\main.dart on AIN065 in debug mode...

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flutter_glyph_kit:compileDebugKotlin'.
> Error while evaluating property 'filteredArgumentsMap' of task ':flutter_glyph_kit:compileDebugKotlin'.
   > Could not resolve all files for configuration ':flutter_glyph_kit:debugCompileClasspath'.
      > Failed to transform KetchumSDK_Community_20231123.jar to match attributes {artifactType=android-classes-jar, org.gradle.libraryelements=jar, org.gradle.usage=java-runtime}.
         > Execution failed for JetifyTransform: C:\Users\jyoth\AppData\Local\Pub\Cache\hosted\pub.dev\flutter_glyph_kit-0.1.1\android\src\lib\KetchumSDK_Community_20231123.jar.
            > Transform's input file does not exist: C:\Users\jyoth\AppData\Local\Pub\Cache\hosted\pub.dev\flutter_glyph_kit-0.1.1\android\src\lib\KetchumSDK_Community_20231123.jar. (See https://issuetracker.google.com/issues/158753935)

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 6s

┌─ Flutter Fix ─────────────────────────────────────────────────────────────┐
│ This issue appears to be https://github.com/flutter/flutter/issues/58247. │
│ Fix this issue by adding the following to the file                        │
│ E:\Projects\glyph_app2\android\app\build.gradle:                          │
│ android {                                                                 │
│   lintOptions {                                                           │
│     checkReleaseBuilds false                                              │
│   }                                                                       │
│ }                                                                         │
└───────────────────────────────────────────────────────────────────────────┘
Error: Gradle task assembleDebug failed with exit code 1

Exited (1).
```

And yes, I've tried doing what they mentioned in the "Flutter Fix" box and the situation is still the same. I'm not that experienced with flutter and I'm just starting out btw.

**Comment by EverythingSuckz** [comment]
It shows that the file name is different. The one in log says it's looking for `KetchumSDK_Community_20231123.jar` but in the path, the name of the file is `KetchumSDK_Community_20240307.jar`. I tried to rename and try but again another error came up. Seems like the [2a update](https://github.com/Nothing-Developer-Programme/Glyph-Developer-Kit/commit/9f6a30cddc387926df31110b0c55c33b6ac41e88#diff-2f4298535d683dec7de25a1690e75da3e7565b8f2f5e3dd01809d98f9adfdf5d) to the SDK file breaks the package?

After renaming the file and debugging the app, I got this:
```
Launching lib\main.dart on AIN065 in debug mode...
e: C:\Users\jyoth\AppData\Local\Pub\Cache\hosted\pub.dev\flutter_glyph_kit-0.1.1\android\src\main\kotlin\dev\francescovallone\glyph_interface_sdk\GlyphInterfaceSdkPlugin.kt: (45, 15): Expecting an element
e: C:\Users\jyoth\AppData\Local\Pub\Cache\hosted\pub.dev\flutter_glyph_kit-0.1.1\android\src\main\kotlin\dev\francescovallone\glyph_interface_sdk\GlyphInterfaceSdkPlugin.kt: (45, 16): Expecting ')
e: C:\Users\jyoth\AppData\Local\Pub\Cache\hosted\pub.dev\flutter_glyph_kit-0.1.1\android\src\main\kotlin\dev\francescovallone\glyph_interface_sdk\GlyphInterfaceSdkPlugin.kt: (42, 9): Expression '(Common.is20111() || Common.is22111() || Common.is23111())' of type 'Boolean' cannot be invoked as a function. The function 'invoke()' is not found

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flutter_glyph_kit:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details
```

**Comment by francescovallone** [comment]
It seems like it, in the weekend I will analyze the problem further and come up with a solution. I'm really sorry that you experienced this error...

**Comment by EverythingSuckz** [comment]
> It seems like it, in the weekend I will analyze the problem further and come up with a solution. I'm really sorry that you experienced this error...

Nah don't be sorry, take your time. It's open source after all. I somehow made it work using the previous version. Thank you so much for this package  ❤️

**Comment by francescovallone** [comment]
Hello, I published version 0.1.2 which contains the fix for this error. Let me know if you are experiencing any issues with the new version 😄.

**Comment by EverythingSuckz** [comment]
Sorry for the late response but yeah it works. Thank you for your time 😄
